broker/ga4gh/broker/server.py

```diff
@@ -0,0 +1,215 @@
+"""Controllers for broker endpoints"""
+
+from flask import request
+from foca.utils.logging import log_traffic
+from werkzeug.exceptions import NotFound
+
+from broker.ga4gh.broker.endpoints.builds import (
+    build_completed,
+    get_build_info,
+    get_builds,
+    register_builds
+)
+from broker.ga4gh.broker.endpoints.repositories import (
+    delete_repository,
+    get_repositories,
+    get_repository_info,
+    modify_repository_info,
+    register_repository
+)
+from broker.ga4gh.broker.endpoints.subscriptions import (
+    delete_subscription,
+    get_subscription_info,
+    get_subscriptions,
+    register_subscription,
+)
+from tests.ga4gh.mock_data import MOCK_SUBSCRIPTION
```

Comment:
Why are we using mocks inside our main server file?

Reply:
I still need to work on `modifySubscription()`. For now, it's just returning `MOCK_SUBSCRIPTION` I'll update it.

Reply:
Yes, please do

Reply:
If you do not want to include subscriptions in this PR (which is totally fine), create a separate branch where you leave this in, then clean up _this_ branch by removing all subscription-related artifacts (starting with the OpenAPI schema, controllers, tests etc.) so that only tested, functional code is in this PR